moveit_msgs/RobotTrajectory trajectory
---
std_msgs/Bool success
